proposals/NNNN-multipart-kit.md

```diff
@@ -0,0 +1,108 @@
+# MultipartKit
+
+* Proposal: [SSWG-NNNN](NNNN-multipart-kit.md)
+* Authors: [Siemen Sikkema](https://github.com/siemensikkema), [Tim Condon](https://github.com/0xTim)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+* Implementation: [repo name](https://github.com/vapor/multipart-kit)
+* Forum Threads: [Pitch](https://forums.swift.org/t/multipartkit), [Discussion](https://forums.swift.org/), [Review](https://forums.swift.org/)
```

Comment:
Given there are no discussion or review thread, we can remove these. I guess a proposal thread will be added when it goes up.